src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+
+      <div class="header__logo"></div>
+
+      <nav class="navigation">
+        <ul class="navigation__items-list">
+          <li class="navigation__items-list__item">
```

Comment:
Wrong BEM usage. Change it to e.g.:
```suggestion
          <li class="navigation__items-list-item">
```
see explanation:

> What would be a class name for an element inside another element? .block__el1__el2 ?

in [https://getbem.com/faq/](https://getbem.com/faq/)